charts/apisix/values.yaml

```diff
@@ -303,14 +303,20 @@ discovery:
       #     send: 2000
       #     read: 5000
 
-# access log and error log configuration
-logs:
```

Comment:
This is a broken change, I suggest not introducing it directly.

Reply:
+1

Reply:
@PGDream plz take a look